ofrak_core/ofrak/core/complex_block.py

```diff
@@ -89,7 +93,7 @@ async def get_mode(self) -> InstructionSetMode:
         :raises ValueError: if the basic blocks in the complex block have more than one mode
         :return: the mode of the complex block
         """
-        await self.resource.unpack()
```

Comment:
As I note below, I don't think you need identifiers in the PR; if they are removed, this change is probably not necessary.